pkg/driverbuilder/builder/ubuntu_test.go

```diff
@@ -229,7 +229,7 @@ var tests = []struct {
 			flavor      string
 			err         error
 		}{
-			headersURLs: []string{"https://mirrors.edge.kernel.org/ubuntu/pool/main/l/linux-kvm/linux-headers-5.19.0-1006-kvm_5.19.0-1006.6_amd64.deb", "https://mirrors.edge.kernel.org/ubuntu/pool/main/l/linux-kvm/linux-kvm-headers-5.19.0-1006_5.19.0-1006.6_all.deb"},
+			headersURLs: []string{},
```

Comment:
Mmmh nice shortcut ahah but i think you need to update/fix headerURLs instead of truncating them :P 

Moreover, the ubuntu tests are really flaky because they depend upon packages provided by ubuntu (ie: headers), but ubuntu is pretty quick in upgrading its supported kernels, and like monthly these tests must be fixed...it would be great to find a way to fix them in a way that aren't going to break again.
I think they are useful but i don't like flaky tests.

Reply:
You're right about it not being the right answer to test this way. The reason I truncated is because that kernel no longer exists (those URLs now produce 404s). It would be better to just remove the URL checks that are reliant on Ubuntu, but I figured that was outside the scope of this PR. I can go adjust all the tests too if you'd like.

Reply:
I think we can do it in another PR, you are right! Thanks!